wp-content/civi-extensions/goonjcustom/Civi/InstitutionService.php

```diff
@@ -184,13 +197,18 @@ public static function setOfficeDetails($op, $groupID, $entityID, &$params) {
 
     $stateOfficeId = $stateOffice['id'];
 
-    $updateGoonjOffice = Organization::update(FALSE)
+    Organization::update(FALSE)
       ->addValue('Review.Goonj_Office', $stateOfficeId)
-      ->addWhere('id', '=', $contactId)
+      ->addWhere('id', '=', self::$organizationId)
       ->execute();
 
+    if (!$stateId) {
+      \Civi::log()->info('state not found', ['organizationId' => self::$organizationId, 'stateId' => $stateId]);
```

Comment:
these logs are good, where it only logs the value when there is no stateId, and on above on the code you are printing logs everytime when functions runs